packages/odyssey-react/src/components/Form/Form.module.scss

```diff
@@ -11,22 +11,22 @@
  */
 
 .root {
-  max-width: calc(#{$max-line-length} + (2 * #{$spacing-m}));
-  margin-block-start: 0;
-  margin-block-end: $spacing-m;
-  margin-inline: 0;
-  padding-block: $spacing-m;
-  padding-inline: $spacing-m;
+  max-width: calc(var(--MaxWidthBase) + (2 * var(--MaxWidthBuffer)));
```

Comment:
`MaxWidthBuffer` here can reuse `SpacePaddingInline`, as that's what's being accommodated for in this `calc()`. 

`MaxWidthBase` can probably stay as `FontMaxLineLength` since it's always referencing a global token.